frontends/qt/Makefile

```diff
@@ -31,8 +31,15 @@ linux:
 	cp build/assets.rcc build/linux-tmp/
 	cp server/libserver.so build/linux-tmp/lib
 	cp ../../vendor/github.com/breez/breez-sdk-go/breez_sdk/lib/linux-amd64/libbreez_sdk_bindings.so build/linux-tmp/lib
-	cd build/linux-tmp && /opt/linuxdeployqt-continuous-x86_64.AppImage BitBox -bundle-non-qt-libs -unsupported-allow-new-glibc
+	cd build/linux-tmp && /opt/linuxdeployqt-continuous-x86_64.AppImage BitBox \
+		-bundle-non-qt-libs \
+		-unsupported-allow-new-glibc \
+		# Add Wayland libs so the app can run natively on Wayland too.
+		# The linuxdeployqt maintainer unfortunately refuses to support it automatically: https://github.com/probonopd/linuxdeployqt/issues/189
+		# The list of related plugins was found by: `find $(qmake -query QT_INSTALL_PLUGINS) | grep wayland`
+		-extra-plugins=platforms/libqwayland-generic.so,platforms/libqwayland-egl.so,wayland-graphics-integration-client,wayland-decoration-client,wayland-shell-integration
 	cp /usr/lib/x86_64-linux-gnu/nss/* build/linux-tmp/lib
+	cp ../../vendor/github.com/breez/breez-sdk-go/breez_sdk/lib/linux-amd64/libbreez_sdk_bindings.so build/linux-tmp/lib
```

Comment:
added by mistake, the same line exists above too

Reply:
Nice catch! Removed, ty